plume-front/src/main.rs

```diff
@@ -66,18 +66,18 @@ fn main() {
 /// Toggle menu on mobile device
 ///
 /// It should normally be working fine even without this code
-/// But :focus-within is not yet supported by Webkit/Blink
+/// But :focus-within is not yet supported by Webkit
 fn menu() {
-    if let Some(button) = document().get_element_by_id("menu") {
+    if let Some(button) = document().get_element_by_id("menu" onclick="") {
```

Comment:
I am not sure what you wanted to do here, but this syntax is not valid. `get_element_by_id` only takes one string parameters, that is the `id` of the element to select.

If you wanted to select `#menu` element with an empty `onclick` property, you can probably use `document().query_selector("#menu[onclick=""]")` (this functions takes a CSS selector). But I'm not even sure it would work, because I don't think using `add_event_listener` like we do changes `onclick`.

Reply:
Yeah, honestly, I wasn't too sure that would work either. The CSS `cursor: pointer ` attribute should be enough for this. I'll see if that works...